google-cloud-error_reporting/docs/instrumentation.md

```diff
@@ -1,39 +1,165 @@
 # Stackdriver Error Reporting Instrumentation
 
-Then google-cloud-error_reporting gem provides a Rack Middleware class that can easily integrate with Rack based application frameworks, such as Rails and Sinatra. When enabled, it automatically gathers application exceptions from requests and submit the information to the Stackdriver Error Reporting service.  
+The google-cloud-error_reporting gem provides framework instrumentation features
+to make it easy to report exceptions from your application.
 
-On top of that, the google-cloud-error_reporting also implements a Railtie class that automatically enables the Rack Middleware in Rails applications when used.
+## Installation
 
-## Rails Integration
+```
+$ gem install google-cloud-error_reporting
+```
+
+## Quick Start
 
-To use the Stackdriver Error Reporting Railtie for Ruby on Rails applications, simply add this line to config/application.rb:
 ```ruby
-require "google/cloud/error_reporting/rails"
+require "google/cloud/error_reporting"
+ 
+# Insert a Rack Middleware to report unhanded exceptions 
+use Google::Cloud::ErrorReporting::Middleware
+ 
+# Or explicitly submit exceptions
+begin
+  fail "Boom!"
+rescue => exception
+  Google::Cloud::ErrorReporting.report exception
+end
 ```
-Then the library can be configured through this set of Rails parameters in config/environments/*.rb:
+
+## Configuration
+The Stackdriver ErrorReporting instrumentation featuers should work by default 
```

Comment:
Several spelling and wording issues. Try this:

The default configuration enables Stackdriver instrumentation features for most applications running on Google Cloud Platform. If you want to run on a non Google Cloud environment or you want to customize the instrumentation behavior, you can modify the Stackdriver configuration through a single interface: